File: C/58.length-of-last-word.c
int lengthOfLastWord(char *s) {
    int len = 0;
    while (s[len])
        len++;

    int i = len - 1;
    while ((i >= 0) && (s[i] == ' '))
        i--;

    int count = 0;
    while ((i >= 0) && (s[i--] != ' '))
        count++;

    return count;
}

File: 58.length-of-last-word.kt
class Solution {
    fun lengthOfLastWord(s: String): Int {
        var i = s.length - 1
        while ((i >= 0) && (s.get(i) == ' '))
            i--

        var count = 0
        while ((i >= 0) && (s.get(i--) != ' ')) 
            count++

        return count
    }
}